sale_backorder/models/sale.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    last_date_delivered = fields.Datetime(
+        string="Last Date Delivered", compute="_compute_last_date_delivered", store=True
+    )
+    last_bill_date = fields.Datetime(
+        string="Last Bill Date", compute="_compute_last_bill_date", store=True
+    )
+    uigd_value = fields.Monetary(
+        string="UIGD Value", compute="_compute_uigd_value", store=True
+    )
+    bo_value = fields.Monetary(
+        string="Backorder Value", compute="_compute_bo_value", store=True
+    )
+
+    @api.depends("order_line.uigd_qty", "order_line.price_unit")
```

Comment:
```suggestion
    @api.depends("order_line.uigd_value")
```
is more correct as you could have cache problems.